test/studies/shootout/meteor/kbrady/meteor-parallel-alt.chpl

```diff
@@ -358,8 +352,8 @@ proc searchLinearHelper(in board, in pos, in used, in placed,
 //
 record BackoffSpinLock {
   var l: atomic bool,
-      lockAttempts: uint,
-      maxLockAttempts = (2**16-1): uint;
+      lockAttempts = 0,
+      maxLockAttempts = (2**16-1);
```

Comment:
I'm not understanding what you mean by the additional upper bound.  You're not suggesting we'd make 2**63 attempts to take the lock, are you?